### DISCUSSION TITLE
Eliminating notebook auto-pushing to blog

### DISCUSSION THREAD

**Discussion by aspencoyle** [RobertsLab/resources#1285] [Q&A] [answered]
Every time I write a new lab notebook post at https://afcoyle.github.io/notebook/, it gets auto-pushed as an entry in the Bitter Crab blog (https://bittercrab.wordpress.com/). I figure this was probably set up before my time with IFTTT or something similar.

Would it be possible to disable that auto-pushing?

**Comment by sr320** [comment, marked as answer]
Should be disabled now.

On Fri, Sep 10, 2021 at 12:54 PM Aidan Coyle ***@***.***>
wrote:

> Every time I write a new lab notebook post at
> https://afcoyle.github.io/notebook/, it gets auto-pushed as an entry in
> the Bitter Crab blog (https://bittercrab.wordpress.com/). I figure this
> was probably set up before my time with IFTTT or something similar.
>
> Would it be possible to disable that auto-pushing?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/RobertsLab/resources/discussions/1285>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABB4PNZM4SYG4SYGOTCVD7DUBJO4NANCNFSM5D2ACL2Q>
> .
>
-- 

Steven B. Roberts, Associate Professor Associate Director - Graduate
Program Coordinator School of Aquatic and Fishery Sciences University of
Washington Fisheries Teaching and Research (FTR) Building - Office 232 1140
NE Boat Street - Seattle, WA 98105 robertslab.info - ***@***.*** - @sr320
vm:206.866.5141 - cell:360.362.3626 schedule a zoom call:
https://d.pr/gsgxVJ

**Reply by aspencoyle** [reply]
Fantastic, thanks a bunch!
